src/db.js

```
@@ -0,0 +1,13 @@
+const { Sequelize } = require('sequelize');
+
+const client = new Sequelize({
```

This is a clean and standard way to initialize the Sequelize client using environment variables. To make the application more robust, consider adding a function to test the database connection (e.g., using `client.authenticate()`) and calling it when the server starts. This ensures that the application fails fast if it can't connect to the database, providing a clear and immediate error message during startup.